### ISSUE TITLE
Fix position of footer to bottom

### ISSUE THREAD

**Issue by elliottSlee** [bcgov/CONN-CCBC-portal#599]
The footer is visible on the screen at all times. For smaller screens, the sticky footer affects accessibility. The footer should be fixed to the bottom of the page. 🔨 ⬇️

#### Acceptance Criteria
Given that a user is on any page
when they arrive on a page that is taller than their screen height
then they will not see the footer
and when they scroll to the bottom
then they will see the footer

**Comment by elliottSlee** [comment]
@MarsRomer Adding this card to the Sprint 12 Candidates

**Comment by elliottSlee** [comment]
Fixed
